docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -3333,6 +3333,12 @@ Start the sensor:
 
 The DHCP traffic should now be reflected on your PacketFence server.
 
+Switch login access
+~~~~~~~~~~~~~~~~~~~
+
+PacketFence is able to allow the access to a switch with read or write permission. It currently works with Cisco switches and in order to configure the switch you have to follow this guide http://www.cisco.com/c/en/us/support/docs/security-vpn/remote-authentication-dial-user-service-radius/116291-configure-freeradius-00.html.
```

Comment:
PacketFence is able to act as an authentication and authorization service for granting command-line interface (CLI) access to switches. PacketFence currently supports Cisco switches and these must be configured using the following guide: http://www.cisco.com/c/en/us/support/docs/security-vpn/remote-authentication-dial-user-service-radius/116291-configure-freeradius-00.html